tc-report/_site/404.html

```diff
@@ -3,7 +3,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>GC Talent | </title>
+  <title> | </title>
```

Comment:
Not sure this 404.html file is ever served, but can it also use a site variable?

Reply:
It can't be served if you manually navigate to the path but it won't be served by the server for "real" 404s.  It can't actually be localized (I think) because it's not under /en or /fr.  I've gone ahead and removed it totally.
[Remove tc-report index and 404 pages](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4468/commits/423685bf337d2564c36249d70518b1f87b957617)